src/app/info/EditWikiItemView.tsx

```diff
@@ -10,54 +10,54 @@ import supabase from "@/supabaseClient";
 import { logErrorReturnLogId } from "@/logger/logger";
 
 interface EditViewProps {
-    rowRef?: DbWikiRow;
+    rowRef: DbWikiRow;
     setRowRef: (row?: DbWikiRow) => void;
-    setEditModeRef: (editMode: boolean) => void;
+    setIsInEditModeRef: (editMode: boolean) => void;
 }
 
 const deleteWikiItem = async (
-    setEditModeRef: (editMode: boolean) => void,
+    setIsInEditModeRef: (editMode: boolean) => void,
     setRowRef: (row?: DbWikiRow) => void,
     rowRef?: DbWikiRow
 ): Promise<void> => {
     if (rowRef) {
-        const result = confirm("Confirm deletion of this item?");
-        if (result) {
+        const confirmation = confirm("Confirm deletion of this item?");
+        if (confirmation) {
             const response = await supabase
                 .from("wiki")
                 .delete()
                 .match({ wiki_key: rowRef.wiki_key });
             if (response.error) {
-                logErrorReturnLogId("error fetching wiki data", response.error);
+                logErrorReturnLogId("error deleting item", response.error);
```

Comment:
sorry to ask for an edit on this again - could we still specify that its a wiki row we're deleting rather than a possibly too generic 'item'